### ISSUE TITLE
should npm run flow be working out-of-the-box?

### ISSUE THREAD

**Issue by ballPointPenguin** [chentsulin/electron-react-boilerplate#844]
Not sure if others would agree, but it seems to me that in a vanilla instance of this boilerplate (either just cloned, or cloned and ran `npm run cleanup`), the commands `npm run test-all` and by extension `npm run flow` should be green.
Currently (#cc9f85e) `npm run flow` is yielding 12 errors.

**Comment by amilajack** [comment]
Is this happening with the latest commit of master?

**Comment by amilajack** [comment]
Closing this because its probably related to the cleanup script, which seems to be causing a lot of problems:

https://github.com/chentsulin/electron-react-boilerplate/issues/837

**Comment by ballPointPenguin** [comment]
This is still an issue, as of https://github.com/chentsulin/electron-react-boilerplate/commit/9beb1fea5f595b3c2b18711ba0c7afeaaf299444

Let me be more explicit. Steps to reproduce:
```
$ git clone https://github.com/chentsulin/electron-react-boilerplate.git
$ cd electron-react-boilerplate
$ npm install
$ npm run test-all
```

expected result:
tests pass out-of-the-box

actual result:
```
> electron-react-boilerplate@0.10.0 test-all /home/brosas/hub/electron-react-boilerplate
> npm run lint && npm run flow && npm run test && npm run build


> electron-react-boilerplate@0.10.0 lint /home/brosas/hub/electron-react-boilerplate
> eslint --cache --format=node_modules/eslint-formatter-pretty .


> electron-react-boilerplate@0.10.0 flow /home/brosas/hub/electron-react-boilerplate
> flow

app/main.development.js:16
 16:   require('module').globalPaths.push(p); // eslint-disable-line
       ^^^^^^^^^^^^^^^^^ module. Required module not found

app/store/configureStore.production.js:14
 14:   return createStore(rootReducer, initialState, enhancer); // eslint-disable-line
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ function call. Function cannot be called on any member of intersection type
 14:   return createStore(rootReducer, initialState, enhancer); // eslint-disable-line
              ^^^^^^^^^^^ intersection
  Member 1:
   41:   declare function createStore<S, A>(reducer: Reducer<S, A>, enhancer?: StoreEnhancer<S, A>): Store<S, A>;
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ polymorphic type: function type. See lib: flow-typed/npm/redux_v3.x.x.js:41
  Error:
   41:   declare function createStore<S, A>(reducer: Reducer<S, A>, enhancer?: StoreEnhancer<S, A>): Store<S, A>;
                                                                               ^^^^^^^^^^^^^^^^^^^ function type. Callable signature not found in. See lib: flow-typed/npm/redux_v3.x.x.js:41
    4: export type counterStateType = {
                                      ^ object type. See: app/reducers/counter.js:4
  Member 2:
   42:   declare function createStore<S, A>(reducer: Reducer<S, A>, preloadedState: S, enhancer?: StoreEnhancer<S, A>): Store<S, A>;
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ polymorphic type: function type. See lib: flow-typed/npm/redux_v3.x.x.js:42
  Error:
   52:   declare function combineReducers<O: Object, A>(reducers: O): Reducer<$ObjMap<O, <S>(r: Reducer<S, any>) => S>, A>;
                                                                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ property `routing`. Property not found in. See lib: flow-typed/npm/redux_v3.x.x.js:52
    4: export type counterStateType = {
                                      ^ object type. See: app/reducers/counter.js:4

node_modules/babel-plugin-flow-runtime/src/Entity.js:19
 19:   get node (): ? Node {
       ^ Potentially unsafe get/set usage. Getters and setters with side effects are potentially unsafe and disabled by default. You may opt-in to using them anyway by putting `unsafe.enable_getters_and_setters=true` into the [options] section of your .flowconfig.

node_modules/babel-plugin-flow-runtime/src/Entity.js:23
 23:   get scope (): ? Scope {
       ^ Potentially unsafe get/set usage. Getters and setters with side effects are potentially unsafe and disabled by default. You may opt-in to using them anyway by putting `unsafe.enable_getters_and_setters=true` into the [options] section of your .flowconfig.

node_modules/babel-plugin-flow-runtime/src/Entity.js:27
 27:   get isTypeAlias (): boolean {
       ^ Potentially unsafe get/set usage. Getters and setters with side effects are potentially unsafe and disabled by default. You may opt-in to using them anyway by putting `unsafe.enable_getters_and_setters=true` into the [options] section of your .flowconfig.

node_modules/babel-plugin-flow-runtime/src/Entity.js:31
 31:   get isImportedType (): boolean {
       ^ Potentially unsafe get/set usage. Getters and setters with side effects are potentially unsafe and disabled by default. You may opt-in to using them anyway by putting `unsafe.enable_getters_and_setters=true` into the [options] section of your .flowconfig.

node_modules/babel-plugin-flow-runtime/src/Entity.js:35
 35:   get isTypeParameter (): boolean {
       ^ Potentially unsafe get/set usage. Getters and setters with side effects are potentially unsafe and disabled by default. You may opt-in to using them anyway by putting `unsafe.enable_getters_and_setters=true` into the [options] section of your .flowconfig.

node_modules/babel-plugin-flow-runtime/src/Entity.js:39
 39:   get isClassTypeParameter (): boolean {
       ^ Potentially unsafe get/set usage. Getters and setters with side effects are potentially unsafe and disabled by default. You may opt-in to using them anyway by putting `unsafe.enable_getters_and_setters=true` into the [options] section of your .flowconfig.

node_modules/babel-plugin-flow-runtime/src/Entity.js:43
 43:   get isValue (): boolean {
       ^ Potentially unsafe get/set usage. Getters and setters with side effects are potentially unsafe and disabled by default. You may opt-in to using them anyway by putting `unsafe.enable_getters_and_setters=true` into the [options] section of your .flowconfig.

node_modules/babel-plugin-flow-runtime/src/Entity.js:47
 47:   get isGlobal (): boolean {
       ^ Potentially unsafe get/set usage. Getters and setters with side effects are potentially unsafe and disabled by default. You may opt-in to using them anyway by putting `unsafe.enable_getters_and_setters=true` into the [options] section of your .flowconfig.

node_modules/babel-plugin-flow-runtime/src/__tests__/fixtures.js:49
 49:   return [filename, require(`./__fixtures__/${filename}`)];
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ The parameter passed to require() must be a literal string.

node_modules/babel-plugin-flow-runtime/tools/makeFixturesJSON.js:3
  3: const dedent = require('dedent');
                    ^^^^^^^^^^^^^^^^^ dedent. Required module not found


Found 12 errors
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! electron-react-boilerplate@0.10.0 flow: `flow`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the electron-react-boilerplate@0.10.0 flow script 'flow'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the electron-react-boilerplate package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     flow
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs electron-react-boilerplate
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls electron-react-boilerplate
npm ERR! There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/brosas/.npm/_logs/2017-03-27T21_44_38_370Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! electron-react-boilerplate@0.10.0 test-all: `npm run lint && npm run flow && npm run test && npm run build`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the electron-react-boilerplate@0.10.0 test-all script 'npm run lint && npm run flow && npm run test && npm run build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the electron-react-boilerplate package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run lint && npm run flow && npm run test && npm run build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs electron-react-boilerplate
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls electron-react-boilerplate
npm ERR! There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/brosas/.npm/_logs/2017-03-27T21_44_38_382Z-debug.log
```

**Comment by VladimirPal** [comment]
Same thing, npm run flow yields lot of errors

**Comment by ballPointPenguin** [comment]
Request to re-open, @amilajack. This is unrelated to the cleanup script.

**Comment by amilajack** [comment]
I just merged a PR today that fixes this. Would be great if you can clone master and see if this was fixed for you

**Comment by amilajack** [comment]
Closing because this was resolved in `0.11.0`
